apps/hobbies-helsinki/src/common-events/components/category/__tests__/__snapshots__/CategoryFilter.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`matches snapshot 1`] = `
 <div
-  class=""
```

Comment:
This looks worse than it is. I compared the snapshot tests both on monorepo and hobbies. On hobbies this gets the class value but other than that the component is exactly the same and it generates the same html on both monorepo and hobbies. Also classes work when tested on browser on monorepo.